modules/ROOT/partials/configuration/link_default_target.adoc

```diff
@@ -3,17 +3,23 @@
 
 This option allows you to set a default `+target+` value for links when inserting/editing a link via the link dialog. If the value of `+link_default_target+` matches a value specified by the xref:link.adoc#link_target_list[`+link_target_list+`] option, that item will be set as the default item for the targets dropdown in the link dialog.
 
+// TODO: Should this be shown if already on the autolink page?
```

Comment:
Like my previous comment, please log a Jira for this and remove the TODO as it's just going to get missed here.